### PR TITLE
support deleting vpcs that are non-default

### PR DESCRIPTION
**Pull request by ekristen**
This takes the existing VPC code and implements it for non-default per region nuking.

### CODE REVIEW

**Review by bwhaley — COMMENTED** [review]
Changes LGTM!

One request: could you please add VPCs to the list of resources in the README, and also add a subsection under Usage that explains that that default VPCs are excluded when nuking VPCs? I agree with you that excluding them makes sense since we have a dedicated `defaults-aws` command for that. Also, nuking the default VPCs would break some internal use cases we have. But we ought to explain our reasoning in the docs.

Thanks!

**Comment by ekristen** [comment]
@bwhaley changes made as requested.

File: README.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ The currently supported functionality includes:
 - Deleting all SQS queues in an AWS account
 - Deleting all S3 buckets in an AWS account - except for buckets tagged with Key=cloud-nuke-excluded Value=true
 - Deleting all default VPCs in an AWS account
+- Deleting all non-default VPCs in an AWS Account - except for default vps which is handled by default-aws
```
Comment (bwhaley) [inline README.md:29]:
```suggestion
- Deleting VPCs in an AWS Account (except for default VPCs which is handled by the dedicated `defaults-aws` subcommand)
```

**Review by bwhaley — COMMENTED** [review]
Thanks. One fix and I think this is ready to ship.

**Review by bwhaley — APPROVED** [review]
LGTM!

**Comment by bwhaley** [comment]
https://github.com/gruntwork-io/cloud-nuke/releases/tag/v0.7.1

**Comment by ekristen** [comment]
Thanks @bwhaley I've got a number of other PRs open ;-) -- I'll make sure to update the README on each one too.

**Pull request merged**